spec/views/sign_up/recovery_codes/show.html.slim_spec.rb

```diff
@@ -43,4 +21,30 @@
 
     expect(rendered).to have_css('.step-3.active')
   end
+
+  context 'when render has no dependencies' do
```

Comment:
I meant it doesn't rely on any instance variables to render the proper output. I'm open to suggestions on a different context description!